pkg/ports/ports_test.go

```diff
@@ -132,11 +132,45 @@ func TestCollectPorts(t *testing.T) {
 				{TargetPort: 7070, Port: 7000, Hostname: "myapp3.local"},
 			},
 		},
+		{
```

Comment:
Would it be possible to add a test around the same port mappings happening on different hostnames?

Reply:
Done :+1: